.github/workflows/test.yaml

```diff
@@ -49,10 +49,12 @@ jobs:
 
       - name: Environment setup
         uses: ./.github/actions/setup
-        with:
-          gh_token: ${{ secrets.GITHUB_TOKEN }}
-          ib_user: ${{ secrets.IRON_BANK_ROBOT_USERNAME }}
-          ib_password: ${{ secrets.IRON_BANK_ROBOT_PASSWORD }}
+
+      - name: Login to GHCR
+        run: uds run -f tasks/setup.yaml registry-login --with registry=ghcr.io --with registry-username=dummy --with registry-password=${{ secrets.GITHUB_TOKEN }} --with registry-retry-interval=5
+
+      - name: Login to registry1
+        run: uds run -f tasks/setup.yaml registry-login --with registry=registry1.dso.mil --with registry-username=${{ secrets.IRON_BANK_ROBOT_USERNAME }} --with registry-password=${{ secrets.IRON_BANK_ROBOT_PASSWORD }} --with registry-retry-interval=5
```

Comment:
nit: since the default `registry-retry-interval` is 5 we can simplify the commands here and drop it I think?

Reply:
agreed, removed it so it takes the default